src/language_negotiation_lambda.js

```diff
@@ -24,32 +37,97 @@ exports.handler = (event, context, callback) => {
                 sanitizedWeight = parseFloat(weight.substring(2)) || 0.0;
```

Comment:
Why do we call `weight.substring(2)` here? I'm not sure that it's doing what we expect.
```
'1.0'.substring(2) # => '0'
```